parquet/src/main/java/org/apache/iceberg/parquet/ParquetIO.java

```diff
@@ -178,7 +178,7 @@ public boolean readVectoredAvailable(ByteBufferAllocator allocate) {
     @Override
     public void readVectored(List<ParquetFileRange> ranges, ByteBufferAllocator allocate)
         throws IOException {
-      IntFunction<ByteBuffer> delegateAllocate = (allocate::allocate);
```

Comment:
Thanks for fixing this, as a minor point, while we're addressing this can we just inline this as part of the call to `delegate.readVectored`?

Reply:
Sure, updated to inline the variable.